### ISSUE TITLE
Wrong disassembly for ARM vstmia/vldmia

### ISSUE THREAD

**Issue by cmp-** [NationalSecurityAgency/ghidra#2304]
**Describe the bug**
ARM instructions vstmia and vldmia are not disassembled correctly.

**To Reproduce**
Set the language to ARM v7 LE and try to disassemble 82EC 046B 91EC 042B in Thumb mode.
The instructions are decoded as:
```
vstmia r2,{d14}
vldmia r1,{d14,d15,d16,d17,d18,d19,d20,d21}
```

**Expected behavior**
The instructions should be decoded as:
```
vstmia r2,{d6,d7}
vldmia r1,{d2,d3}
```

**Environment (please complete the following information):**
 - OS: Windows 10
 - Java Version: 11.0.8
 - Ghidra Version: 9.2_DEV (master)

**Comment by emteere** [comment]
Good find!
Fixed and will be corrected in 9.2 release and master, most likely in master today

**Comment by ghizard** [comment]
Should be fixed by 42b8eb3
If not, please reopen.
